dataland-frontend/src/components/forms/CreateEuTaxonomyForFinancials.vue

```diff
@@ -516,6 +519,37 @@ export default defineComponent({
         .catch((e) => console.log(e));
     },
 
+    /**
+     * Parses received financial service types and eligibility KPIs data
+     * @param receivedFormInputsModelData Received data
+     * @returns An object with financial service types and eligibility KPIs which can be directly merged into the formInputsModel, which will then populate expected KPI section fields
+     */
+    extractEligibilityKpis(receivedFormInputsModelData: ObjectType) {
+      const kpiKeys = Object.keys(euTaxonomyKPIsModel.kpisFieldNameToFinancialServiceType);
```

Comment:
`kpiKeys` contains the singular of kpi which is not correct. What do you think of `financialServiceTypeKeys`?

Reply:
Done.